seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -83,16 +87,18 @@ def test_anvil_workspace_page(self, mock_logger):
         self.mock_get_ws_access_level.assert_not_called()
 
     @mock.patch('seqr.views.apis.anvil_workspace_api.BASE_URL', 'http://testserver/')
-    @mock.patch('seqr.views.apis.anvil_workspace_api.time.sleep')
+    @mock.patch('seqr.views.apis.anvil_workspace_api.time')
     @mock.patch('seqr.views.apis.anvil_workspace_api.logger')
     @mock.patch('seqr.views.apis.anvil_workspace_api.load_uploaded_file')
     @mock.patch('seqr.views.apis.anvil_workspace_api.has_service_account_access')
     @mock.patch('seqr.views.apis.anvil_workspace_api.add_service_account')
-    @mock.patch('seqr.utils.communication_utils.EmailMultiAlternatives')
+    @mock.patch('seqr.views.apis.anvil_workspace_api.safe_post_to_slack')
     @mock.patch('seqr.views.apis.anvil_workspace_api.does_file_exist')
     @mock.patch('seqr.views.apis.anvil_workspace_api.file_iter')
-    def test_create_project_from_workspace(self, mock_file_iter, mock_file_exist, mock_email, mock_add_service_account,
-                                           mock_has_service_account, mock_load_file, mock_api_logger, mock_sleep,
+    @mock.patch('seqr.views.apis.anvil_workspace_api.mv_file_to_gs')
```

Comment:
so `mv_file_to_gs` is not going to be tested anywhere because you mock it here. So far, we've never explicitly added tests for the file utils file, we test the functions where they are called. For example, if you look at igv_api_tests.py and data_manager_api_tests.py you see it actually them mocking subprocess and checking that the correct gsutil command is being run. We either need to do that here, or we need to add a test file and add an explicit test for this helper method

Reply:
I added a test file to test the helper.